drivers/counter/counter_mcux_gpt.c

```diff
@@ -25,45 +26,51 @@ struct mcux_gpt_config {
 };
 
 struct mcux_gpt_data {
+	DEVICE_MMIO_RAM;
```

Comment:
It seems like this change will move the device pointer into RAM for all devices (not just A cores with an MMU), is this right? I would expect we'd need a `DEVICE_MMIO_ROM` in the config structure in order to keep the device pointer in ROM for those parts.

Reply:
Nevermind- answered my own question, we do have a `DEVICE_MMIO_ROM` being declared in the device init macro